### ISSUE TITLE
[A11y] - `<SideBar />` items lack 3 to 1 contrast ratio (for hover/focus)

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#587]
<dl><dt>Description</dt><dd> The state of an ACTIVE user interface component does not have at least a 3 to 1 luminosity contrast ratio with either the inner or outer adjacent background.
Refer to the focus indicator color of the left navigational links.

Note: This issue is for all Nav items when they receive keyboard focus. 

Focus Indicator color::#F2F2F2
Adjacent color:#CCCCCC
Color contrast: 1.4:1</dd><dt>Impact</dt><dd>Serious</dd><dt>Recommendation</dt><dd>RULE : 
The visual state of an active user interface component MUST have sufficient contrast of 3 to 1 with the adjacent background. Exceptions exist.

HOW TO FIX: 
Fix this issue by adjusting the visual focus indicator or state indicator (e.g. selected, checked, pressed, etc.) of the user interface component and/or background to increase the contrast with either the inner or outer adjacent background to at least 3 to 1.

REFERENCE: 
WCAG Understanding document: https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast

BACKGROUND: 
People who have low vision or are colorblind may have difficulty perceiving that an element is interactive or what its state is - e.g. whether it has keyboard focus, is selected/checked/pressed, etc. - if the contrast between the element boundaries and/or state indicators and its background or adjacent colors is insufficient. When an interactive element's visual boundaries and state indicators have adequate contrast, people who have low vision or are colorblind are more likely to be able to perceive which elements on the screen are interactive and what their current state is.</dd><dt>Source code</dt><dd>
```html
<nav><ul class="SideBar"><li class="MenuItem--active"><a aria-current="page" href="/components/Address">Address</a></li><li class=""><a href="/components/Alert">Alert</a></li><li class=""><a href="/components/Breadcrumb">Breadcrumb</a></li><li class=""><a href="/components/Button">Button</a></li><li class=""><a href="/components/Card">Card</a></li><li class=""><a href="/components/Checkbox">Checkbox</a></li><li class=""><a href="/components/ClickOutsideListener">ClickOutsideListener</a></li><li class=""><a href="/components/Code">Code</a></li><li class=""><a href="/components/DescriptionList">DescriptionList</a></li><li class=""><a href="/components/ExpandCollapsePanel">ExpandCollapsePanel</a></li><li class=""><a href="/components/FieldWrap">FieldWrap</a></li><li class=""><a href="/components/Icon">Icon</a></li><li class=""><a href="/components/IconButton">IconButton</a></li><li class=""><a href="/components/IssuePanel">IssuePanel</a></li><li class=""><a href="/components/Layout">Layout</a></li><li class=""><a href="/components/Line">Line</a></li><li class=""><a href="/components/Link">Link</a></li><li class=""><a href="/components/Loader">Loader</a></li><li class=""><a href="/components/LoaderOverlay">LoaderOverlay</a></li><li class=""><a href="/components/Modal">Modal</a></li><li class=""><a href="/components/NavBar">NavBar</a></li><li class=""><a href="/components/OptionsMenu">OptionsMenu</a></li><li class=""><a href="/components/Pagination">Pagination</a></li><li class=""><a href="/components/Panel">Panel</a></li><li class=""><a href="/components/Pointout">Pointout</a></li><li class=""><a href="/components/ProgressBar">ProgressBar</a></li><li class=""><a href="/components/RadioGroup">RadioGroup</a></li><li class=""><a href="/components/Select">Select</a></li><li class=""><a href="/components/Stepper">Stepper</a></li><li class=""><a href="/components/Table">Table</a></li><li class=""><a href="/components/Tabs">Tabs</a></li><li class=""><a href="/components/Tag">Tag</a></li><li class=""><a href="/components/TextField">TextField</a></li><li class=""><a href="/components/Toast">Toast</a></li><li class=""><a href="/components/Tooltip">Tooltip</a></li><li class=""><a href="/components/TooltipTabstop">TooltipTabstop</a></li><li class=""><a href="/components/TopBar">TopBar</a></li><li class=""><a href="/components/TopBarMenu">TopBarMenu</a></li></ul></nav>
```
</dd><dt>Checkpoint</dt><dd>Non-Text Contrast - States of User Interface Components (1.4.11.b)</dd><dt>URL</dt><dd>https://cauldron.dequelabs.com/</dd></dl>

**Comment by okry** [comment]
@awpearlm Do we have anywhere on the uxpin that has colors for these?

**Comment by scurker** [comment]
It looks like it's just the focus state that needs indication, it doesn't look like there's a specific requirement for hover:

> For example, there is not a new requirement that visited links contrast with the default color, or that mouse hover indicators contrast with the default state.

We also need to determine if anyone is actually using this. We need to ask if other teams have this in use, if not we should remove it as a pattern. It's not in the uxpin and only appears to be in the cauldron docs.

**Comment by scurker** [comment]
This is not a pattern that is present in Cauldron documentation. Closing.
